lldb/test/API/lang/swift/async/stepping/step-in/task-switch/TestSwiftTaskSwitch.py

```diff
@@ -15,11 +15,25 @@ def test(self):
         target, _, thread, _ = lldbutil.run_to_source_breakpoint(self, "await f()", src)
         self.assertEqual(thread.frame[0].function.mangled, "$s1a5entryO4mainyyYaFZ")
 
-        function = target.FindFunctions("$s1a5entryO4mainyyYaFZTQ0_")[0].function
-        instructions = list(function.GetInstructions(target))
+        sym_ctx_list = target.FindFunctions("$s1a5entryO4mainyyYaFZTQ0_", lldb.eFunctionNameTypeAuto)
+        self.assertEquals(sym_ctx_list.GetSize(), 1)
+        symbolContext = sym_ctx_list.GetContextAtIndex(0)
```

Comment:
for consistency, could you name this `symbol_context` or `symbol_ctx`, thanks.

Reply:
Sorry, my mistake.
I will update if this PR is still needed.